package.json

```diff
@@ -21,7 +21,11 @@
     "kibble:build": "kibble publish",
     "build": "npm-run-all css js:build css:build kibble:build",
     "language:csv": "node scripts/language-file-to-csv/language-file-to-csv.js site/en_AU.all.json",
-    "language:validate": "node scripts/language-file-validator.js"
+    "language:validate": "node scripts/language-file-validator.js",
+    "version": "node scripts/core-template-version.js",
+    "prestart": "npm run version",
+    "prebuild": "npm run version",
+    "prepare": "npm run version"
```

Comment:
Just noting this from the docs:

> As of npm@7 these scripts run in the background. To see the output, run with: --foreground-scripts.

Do you see output?

Reply:
I don't see anything out of the ordinary adding `--foreground-scripts` but I do see the logging from `core-template-version.js` so the `pre` scripts are working fine.

Reply:
Can you verify you are on v7+?

Reply:
I'm on 8.1.1